test/AnchorRepository.test.js

```diff
@@ -117,7 +117,9 @@ contract("AnchorRepository", function (accounts) {
             assert.equal(bufferToHex(hashedAnchorId), web3.utils.toHex(response[0]));
             assert.equal(documentRoot, response[1]);
 
-
+            let blckNoResponse = await this.anchorRepository.getAnchoredBlockById.call(hashedAnchorId, callOptions);
+            assert.equal(bufferToHex(hashedAnchorId), web3.utils.toHex(blckNoResponse[0]));
+            assert.isAbove(0,  web3.utils.BN(blckNoResponse[1]));
```

Comment:
@rdinicut whats the best way to verify this block number?

Reply:
Why do we need bytes32 instead of uint for the block number?